packages/clay-drop-down/src/DropDownWithItems.tsx

```diff
@@ -140,6 +140,11 @@ export interface IProps
 
 	menuWidth?: React.ComponentProps<typeof ClayDropDown>['menuWidth'];
 
+	/**
+	 * Prop to use language keys.
+	 */
+	messages?: string;
```

Comment:
Can we make this prop `ariaLabel`? I think it's a clearer name for it.

Reply:
Hey @pat270, the reason I added this was because we have the `messages` prop in other components that have the same purpose, add language keys.

Reply:
I see.